src/modules/fw_att_control/fw_att_control_params.c

```diff
@@ -96,6 +96,21 @@ PARAM_DEFINE_FLOAT(FW_P_TC, 0.4f);
  */
 PARAM_DEFINE_FLOAT(FW_PR_P, 0.08f);
 
+/**
+ * Pitch rate derivative gain.
+ *
+ * This defines how much the elevator input will be commanded depending on the
```

Comment:
Not sure if this explanation is helpful, I would rather go with something like for the MC D terms, so:

Pitch rate differential gain. Small values help reduce fast oscillations. If value is too big oscillations will appear again.

For sure try to avoid naming specific actuators here, as there doesn't need to be an elevator controlling the pitch of a vehicle.

Reply:
Done!